front/app/components/admin/PostManager/ProjectProposalsManager.tsx

```diff
@@ -54,7 +54,9 @@ const ProjectProposalsManager = ({
     useState<TFilterMenu>(defaultFilterMenu);
   const [previewPostId, setPreviewPostId] = useState<string | null>(null);
   const [previewMode, setPreviewMode] = useState<PreviewMode>('view');
-  const { data: initiativeStatuses } = useInitiativeStatuses();
+  const { data: ideaStatuses } = useIdeaStatuses({
+    participation_method: 'proposals',
+  });
```

Comment:
Should we call this `proposalStatuses` instead?

Reply:
Or I also see inputStatuses in some contexts - not quite sure what you've settled on for new terms? :)

Reply:
@amanda-anderson We're indeed a bit in a confusing situation, like in the backend there are idea/ideation statuses for ideation, proposals statuses for proposals, input statuses to refer to the combination of both and initiative statuses which is the old implementation we still need to keep for now 😅 We're hoping to unify the idea and proposals statuses to one input statuses system at some point.

For the backend (I suspect this is also applicable for the frontend), we decided to ignore this situation for now. We're often using the term "input" instead of "idea" in new code, and there's a separate task to rename "idea" to "input" throughout the codebase (including endpoints and database migrations, which takes some effort). When deleting the old code, we will also get rid of occurrences of "initiative".

So the ideal situation will be to use "input" almost everywhere, and perhaps occasionally use "idea" or "proposal" when we're in a place where we know that we're dealing with this specific type of input.

Reply:
Ok clear - thanks Sebi! :)

Reply:
Maybe I can give it a try to rename `useIdeaStatuses` to `useInputStatuses`?

Reply:
I think it's ok to leave it for now :) There are other hooks like `useAddIdeaStatus` for instance which would ideally be re-named as well. Better & easier to convert them altogether at some point in the future I think! :+1: